src/RA_AchievementOverlay.cpp

```diff
@@ -11,9 +11,11 @@
 #include "services\ILeaderboardManager.hh"
 #include "services\ServiceLocator.hh"
 
+#ifndef PCH_H
 #include <fstream>
 #include <memory>
-#include <ctime>
+#include <ctime>  
+#endif /* !PCH_H */
```

Comment:
Are these chunks still necessary if `pch.h` is being included by both projects?

Reply:
Well not really, I was afraid if a different project in the future didn't have pch we wouldn't have to adjust for it. I'll get rid of them.